admin_guide/managing_networking.adoc

```diff
@@ -306,29 +306,25 @@ link:http://pubs.vmware.com/vsphere-4-esxi-installable-vcenter/index.jsp?topic=/
 Transmissions] guidance.
 
 [[admin-guide-egress-router-modes]]
-==== Egress Router Modes
+*Egress Router Modes*
 
-The egress router can run in two different modes:
+The egress router can run in two different modes: xref:admin-guide-deploying-an-egress-router-pod[redirect mode] and xref:admin-guide-deploying-an-egress-router-http-proxy-pod[HTTP proxy mode].
 
-1. A "redirect" mode, where the egress router sets up iptables rules
-to redirect traffic from its own IP address to one or more destination
-IP addresses. Client pods that want to make use of the reserved source
-IP address will need to be modified to connect to the egress router
-rather than connecting directly to the destination IP.
+Use HTTP proxy mode for HTTP and HTTPS services. For all other types of connections, redirect mode is recommended.
```

Comment:
It's not just "recommended"; the HTTP proxy mode won't work at all for non-HTTP-based services